### PR TITLE
Add Qualification class to applicant

### PR DESCRIPTION
**Pull request by e0543517**

### CODE REVIEW

**Review by Justinnje — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/model/applicant/Qualification.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.applicant;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's name in the address book.
```
Comment (seanleong339) [inline src/main/java/seedu/address/model/applicant/Qualification.java:7]: Should be Applicant's Qualification here?

**Review by seanleong339 — CHANGES_REQUESTED** [review]
Other than minor javadoc error, LGTM. Will approve and merge after the change

File: src/main/java/seedu/address/model/applicant/Qualification.java (hunk 1 above)
Comment (e0543517) [reply src/main/java/seedu/address/model/applicant/Qualification.java]: Fixed the error, please review again.

**Review by seanleong339 — APPROVED** [review]
LGTM

**Pull request merged**